### PR TITLE
RW-31 admin theme

### PR DESCRIPTION
**Pull request by left23**
I've merged https://github.com/UN-OCHA/rwint9-site/pull/74 into this branch for the Theme Switcher module.

Also includes:

- local tasks styles
- Form styles (currently added as part of global library)
- Template overrides for Node edit forms and Term edit forms started - general method arrived at from the discussion on #74 , most markup present but specific fields and general details have not been covered (i.e. everything is a mess)
- Admin menu component (menu items currently hard coded)
- RW Headlines component with JS in place (needs to be ported to function)
- User pages started - templates in place and some styles copied from D7 (Subscription form, Dashboard template)

Currently using https://github.com/UN-OCHA/common_design/pull/254 base theme branch `cd-304-forms`

### CODE REVIEW

**Comment by left23** [comment]
@orakili To review at your leisure. Note I've been using the CD branch https://github.com/UN-OCHA/common_design/pull/254 to test generic form styles too, which is still a WIP.

If possible, try to ignore all the styling flaws (otherwise you might be making JIRA issues for days).

Once this is merged, I'll create a ticket per Trello card in the Backend:Pages as I work on the details of each.

File: html/themes/custom/common_design_subtheme/templates/user/admin-menu.html.twig [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+          <li>
+            <h3 id="admin-menu-create">Create</h3>
+            <ul aria-labelledby="admin-menu-create">
+              <li><a href="/node/add/report">Report</a></li>
+              <li><a href="/node/add/job">Job</a></li>
+              <li><a href="/node/add/training">Training</a></li>
+              <li><a href="/node/add/topics">Topic</a></li>
+              <li><a href="/admin/structure/taxonomy/country/add">Country</a></li>
+              <li><a href="/admin/structure/taxonomy/disaster/add">Disaster</a></li>
+              <li><a href="/admin/structure/taxonomy/source/add">Source</a></li>
```
Comment (orakili) [inline html/themes/custom/common_design_subtheme/templates/user/admin-menu.html.twig:23]: Looks like the urls to create terms have change in D8/9 to `admin/structure/taxonomy/manage/{vocabulary}/add`

```suggestion
              <li><a href="/admin/structure/taxonomy/manage/country/add">Country</a></li>
              <li><a href="/admin/structure/taxonomy/manage/disaster/add">Disaster</a></li>
              <li><a href="/admin/structure/taxonomy/manage/source/add">Source</a></li>
```

File: html/themes/custom/common_design_subtheme/templates/user/admin-menu.html.twig [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+              <li><a href="/node/add/blog-post">Blog post</a></li>
```
Comment (orakili) [inline html/themes/custom/common_design_subtheme/templates/user/admin-menu.html.twig:24]:
```suggestion
              <li><a href="/node/add/blog_post">Blog post</a></li>
```

File: html/themes/custom/common_design_subtheme/templates/user/user-dashboard.html.twig [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    <li>
+      <a href="/node/add/training">{% trans %}Post a training program{% endtrans %}</a>
+      <p>{% trans %}Advertise training programs for the humanitarian community.{% endtrans %}</p>
+    </li>
+    <li>
+      <a href="{{ url }}/user/posts">{% trans %}View your posts{% endtrans %}</a>
+      <p>{% trans %}Manage the Job vacancies or Training programs you posted.{% endtrans %}</p>
+    </li>
+  </ul>
+</nav>
```
Comment (orakili) [inline html/themes/custom/common_design_subtheme/templates/user/user-dashboard.html.twig:29]:
```suggestion
<nav id="user-dashboard" aria-labelledby="user-dashboard-title" class="rw-user-dashboard">
  <h3 id="user-dashboard-title" class="visually-hidden">{% trans %}Dashboard{% endtrans %}</h3>
  <ul>
    <li>
      <a href="/user//{{ user.id() }}/edit">{% trans %}Edit your account details{% endtrans %}</a>
      <p>{% trans %}Edit your credentials and account information.{% endtrans %}</p>
    </li>
    <li>
      <a href="/user/{{ user.id() }}/notifications">{% trans %}Manage your subscriptions{% endtrans %}</a>
      <p>{% trans %}Setup and manage your email subscriptions to new reports, jobs and more.{% endtrans %}</p>
    </li>
    <li>
      <a href="/user/{{ user.id() }}/bookmarks">{% trans %}Manage your bookmarks{% endtrans %}</a>
      <p>{% trans %}Track content relevant to you on the site.{% endtrans %}</p>
    </li>
    <li>
      <a href="/node/add/job">{% trans %}Post a job vacancy{% endtrans %}</a>
      <p>{% trans %}Advertise job, consulting and internships vacancies.{% endtrans %}</p>
    </li>
    <li>
      <a href="/node/add/training">{% trans %}Post a training program{% endtrans %}</a>
      <p>{% trans %}Advertise training programs for the humanitarian community.{% endtrans %}</p>
    </li>
    <li>
      <a href="/user/{{ user.id() }}/posts">{% trans %}View your posts{% endtrans %}</a>
      <p>{% trans %}Manage the Job vacancies or Training programs you posted.{% endtrans %}</p>
    </li>
  </ul>
</nav>
```

File: html/themes/custom/common_design_subtheme/templates/user/user.html.twig [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+
+  {% if logged_in %}
+    {% include '@common_design_subtheme/user/user-dashboard.html.twig' %}
+  {% endif %}
+
+  {% if content %}
+    {{- content -}}
+  {% endif %}
+</article>
```
Comment (orakili) [inline html/themes/custom/common_design_subtheme/templates/user/user.html.twig:32]: I think we could replace the content of this template with the content of `user-dashboard.html.twig` as that's all we want to display. What do you think?

**Review by orakili — APPROVED** [review]
It's good base to move forward, thanks!

I fixed a few links in the admin menu and user dashboard and created/linked to some tickets for the headline widget, copyright in footer, admin menu permission and consolidating the form suggestions.

Merging.

**Review by orakili — DISMISSED** [review]

**Review by orakili — APPROVED** [review]

**Pull request merged**